### PR TITLE
Added info for imguralbum, aria2, webui-aria2, and youtube-dl

### PR DESCRIPTION
**Pull request by Caleb68864**

### CODE REVIEW

File: video/youtube-dl/README.md [hunk 1]
```diff
@@ -1,4 +1,12 @@
-# youtube-dl
+## youtube-dl
```
Comment (matthiasbeyer) [inline video/youtube-dl/README.md]: Why did you move this to the second level?

File: video/youtube-dl/README.md [hunk 2]
```diff
@@ -1,4 +1,12 @@
...
 
-* Homepage: _insert homepage link here_
+##### Homepage: https://rg3.github.io/youtube-dl/
```
Comment (matthiasbeyer) [inline video/youtube-dl/README.md]: Why not using the old version?

File: video/youtube-dl/README.md [hunk 3]
```diff
@@ -1,4 +1,12 @@
...
+
+youtube-dl is a command-line program to download videos from YouTube.com and a few more sites. 
+It requires the Python interpreter (2.6, 2.7, or 3.2+), and it is not platform specific. 
+We also provide a Windows executable that includes Python. 
+youtube-dl should work in your Unix box, in Windows or in Mac OS X. 
+It is released to the public domain, which means you can modify it, redistribute it or use it however you like.
```
Comment (matthiasbeyer) [inline video/youtube-dl/README.md]: I really like that you wrote one sentence per line! I do that too, so nothing new for me, but :+1: for doing so!

File: web/websites/imguralbum/README.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
+## imguralbum
+
+##### Homepage: https://github.com/alexgisby/imgur-album-downloader
+
+Tool for downloading Imgur Albums with python
+
+* Install for Linux
+		
+		Run install_ImgurAlbum.sh
```
Comment (matthiasbeyer) [inline web/websites/imguralbum/README.md]: Why this huge indentation here?

Comment (matthiasbeyer) [inline web/websites/imguralbum/README.md]: Ah, I see you refer to the .sh file you committed. I would actually include this code snippet in the md file,...

File: video/youtube-dl/README.md (hunk 1 above)
Comment (Caleb68864) [inline video/youtube-dl/README.md]: No real reason just not super familiar with the github readme formatting.
And was trying to figure how to best format it.

A template for how you want format the read me's would very helpful.
At least for me anyway.


On Thu, Jun 30, 2016 at 3:25 PM, Matthias Beyer <notifications@github.com>
wrote:

> In video/youtube-dl/README.md
> <https://github.com/matthiasbeyer/awesome-cli/pull/4#discussion_r69203819>
> :
>
> > @@ -1,4 +1,12 @@
> > -# youtube-dl
> > +## youtube-dl
>
> Why did you move this to the second level?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/matthiasbeyer/awesome-cli/pull/4/files/f22713d719462e6ddfbce604301f2546b7cb9473#r69203819>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABPpwa3np-vDS6Uikdd8zF5dF280nKkNks5qRCYdgaJpZM4JCcs9>
> .
>



-- 
-Caleb Bennett
"The Price Of Freedom Is The Willingness To Do Sudden Battle Anywhere, Any
Time And With Utter Recklessness" - R.A.Heinlein
"An armed society is a polite society." - R.A.Heinlein

**Comment by matthiasbeyer** [comment]
Thanks for your PR! Awesome! I didn't expect to get PRs on the first date this repository is up, so thank you a lot!

File: video/youtube-dl/README.md (hunk 2 above)
Comment (Caleb68864) [inline video/youtube-dl/README.md]: Same as above trying to figure out Read Me formatting.

File: video/youtube-dl/README.md (hunk 3 above)
Comment (Caleb68864) [inline video/youtube-dl/README.md]: Thanks

File: video/youtube-dl/README.md (hunk 1 above)
Comment (matthiasbeyer) [inline video/youtube-dl/README.md]: Nice suggestion. Will add this to the issues.

File: web/websites/imguralbum/README.md (hunk 4 above)
Comment (Caleb68864) [inline web/websites/imguralbum/README.md]: Is there a specific way to include it as a code snippet?
Or do I just need to add it to the readme?

Comment (matthiasbeyer) [inline web/websites/imguralbum/README.md]: I don't think there is a way to do this, so just include it. As long as it is less than 15 lines which are not too complex, I'm fine with it.

**Comment by matthiasbeyer** [comment]
Are you ready? I would merge this now (or rather tomorrow, as I'm going to bed now).

**Comment by Caleb68864** [comment]
I'm ready yes
